### PR TITLE
- add PacManTool for pacman (Arch Linux)

### PR DESCRIPTION
**Pull request by SSE4**
add PacManTool as SystemPackageTool implementation for Arch Linux
also could be used for MSYS2 (but I don't know how to detect MSYS2 environment ATM, and not sure if it is a sane default)

### CODE REVIEW

**Comment by lasote** [comment]
Thanks! good addition.
It's ok for now but I'm adding an issue to add end2end tests to our release infrastructure to test the system package tools with docker to be sure that they are working good.

File: conans/client/tools/oss.py [hunk 1]
```diff
@@ -100,6 +100,11 @@ def with_yum(self):
                                  ("centos", "redhat", "fedora", "pidora", "scientific",
                                   "xenserver", "amazon", "oracle", "rhel")
 
+    @property
+    def with_pacman(self):
+        return self.is_linux and self.linux_distro in \
```
Comment (lasote) [inline conans/client/tools/oss.py]: Same line, and probably == is enough

Comment (SSE4) [reply conans/client/tools/oss.py]: fixed, thanks

**Comment by memsharded** [comment]
Maybe @tivek wants to have a look to this PR

**Comment by tivek** [comment]
This PR looks ok.

There is a cultural issue we might need to keep an eye on. Arch is a rolling release so in principle unattended upgrades and installs on production systems are not recommended. I am sure most Conan users on Arch will know to be careful, but maybe we should consider documenting this or printing a warning.

**Comment by memsharded** [comment]
Ok, despite the minor issues or missing things like docs or e2e tests, I think we can merge it and release in 0.28, so it can be already tested by users.

**Pull request merged**